cql3/Cql.g

```diff
@@ -1748,18 +1745,8 @@ tupleOfTupleLiterals returns [std::vector<expression> literals]
     : '(' t1=tupleLiteral { $literals.emplace_back(std::move(t1)); } (',' ti=tupleLiteral { $literals.emplace_back(std::move(ti)); })* ')'
     ;
 
-markerForTuple returns [expression marker]
-    : QMARK { $marker = new_bind_variables(nullptr); }
```

Comment:
This calls the constructor a bit differently from your new "marker" (it passes nullptr instead of {}), I trust that you checked that both of them do the same thing.

Reply:
Yes, I see it's just a shared_ptr. Good.